### PR TITLE
midori: update to 6.0.

### PR DESCRIPTION
**Pull request by m3tav3rse**

### CODE REVIEW

**Comment by maxice8** [comment]
request-for-review: @Cogitri

**Comment by Cogitri** [comment]
wfm

File: srcpkgs/midori/template [hunk 1]
```diff
@@ -1,31 +1,18 @@
...
-checksum=96191a96be71144ae848a409fae5a1d6d52a00e583f33122081f47ead9c49c3d
-
-pre_build() {
-	sed 's/protected Tally/public Tally/g' -i midori/midori-notebook.vala
-	sed 's/%d other files/%u other files/g' -i extensions/transfers.vala
-	for f in transfers adblock/widgets apps history-list notes; do
-	    sed 's/.remove (iter/.remove (ref iter/g' -i extensions/"${f}".vala
-	done
-}
+license="LGPL-2.1-only"
```
Comment (Cogitri) [inline srcpkgs/midori/template]: version 2.1 of the License, or (at your option) any later version.

Comment (Cogitri) [reply srcpkgs/midori/template]: https://github.com/midori-browser/core/blob/master/web/activatable.vala#L7

File: srcpkgs/midori/template [hunk 2]
```diff
@@ -1,31 +1,18 @@
...
-configure_args="-DUSE_GTK3=1 -DHALF_BRO_INCOM_WEBKIT2=ON
- $(vopt_if zeitgeist '-DUSE_ZEITGEIST=1' '-DUSE_ZEITGEIST=0')"
-hostmakedepends="pkg-config intltool librsvg-utils glib-devel vala-devel libzeitgeist-devel"
-makedepends="gtk+3-devel libxml2-devel vala-devel $(vopt_if zeitgeist libzeitgeist-devel)
- dbus-glib-devel librsvg-devel libnotify-devel libsoup-gnome-devel
- webkit2gtk-devel sqlite-devel libunique-devel libXScrnSaver-devel gcr-devel"
+hostmakedepends="glib-devel gobject-introspection intltool librsvg-utils
+pkg-config vala-devel"
+makedepends="gcr-devel libpeas-devel librsvg-devel libsoup-gnome-devel
+vala-devel webkit2gtk-devel"
```
Comment (Cogitri) [inline srcpkgs/midori/template]: This also depends on gtk-devel and sqlite-devel - but it's pulled in anyway, so I guess it doesn't matter...

Comment (m3tav3rse) [reply srcpkgs/midori/template]: I think  so

**Pull request merged**